### ISSUE TITLE
External link is not shown in the tab nav

### ISSUE THREAD

**Issue by MrF3lix** [squidfunk/mkdocs-material#1884]
## Description

According to the MkDocs docs it is allowed to have external links in the [navigation](https://www.mkdocs.org/user-guide/configuration/#nav). However it is not shown on desktop.
A similar [issue](https://github.com/squidfunk/mkdocs-material/issues/868) has been closed but in my opinion this is not solved.

### Expected behavior
The external navigation item should be shown in the mobile navigation and on the tab navigation on desktop.

### Actual behavior
The external link is only shown in the mobile navigation.

**Mobile:**
![image](https://user-images.githubusercontent.com/7260168/91164722-d7b7ff00-e6cf-11ea-8a41-7d8098603d4f.png)

But not in the desktop navigation: 

**Desktop:**
![image](https://user-images.githubusercontent.com/7260168/91164778-ec949280-e6cf-11ea-9b9f-d47382181e1e.png)

### Steps to reproduce the bug

1. Add an external nav item to the top level navigation.
2. Run `mkdocs serve`
3. Resize window to < 1220px show working mobile nav
4. Resize window to > 1220px to show desktop nav

### Package versions

* Python: `3.8.2`
* MkDocs: `1.1.2`
* Material: `5.5.8`

### Project configuration
Navigation config looks similar to this:

``` yaml
nav:
  - ALM: []
  - DevOps: []
  - UI Styleguide: []
  - Mercury: []
  - SCHS: []
  - Sensors: []
  - Bühler Insights: https://docs.buhlergroup.io

```

### Notes
I think it works in the [`nav-item.html`](https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/nav-item.html#L123) because it renders items that have no child elements.

Where as [`tab-item.html`](https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/tabs-item.html) only renders items that have children or are the main page.

My suggestion is to add this to line 69 in tabs-item.html:

```HTML
{% else %}
  <li class="md-tabs__item">
    <a href="{{ nav_item.url | url }}" class="md-tabs__link">
      {{ nav_item.title }}
    </a>
  </li>
{% endif %}
```
But I'm not sure what the reason is to not have this here?
Let me know if I should create a PR to fix this.

**Comment by squidfunk** [comment]
There are two ways how tabs work, i.e. top-level pages and top-level sections. [The docs](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs) explain it in detail and include an example layout for both configurations. The problem with your suggestion is that the top-level pages approach would not work anymore, as all single-pages will now show up as tabs. I don't know whether it's possible to distinguish external links fro internal Markdown pages, but if that is possible, we can include a branch in `tab-item.html` to include top-level external links.

**Comment by MrF3lix** [comment]
Ok so we would need a way to compare the URL of the nav item with the URL of the page.
Could we use [`page.abs_url`](https://www.mkdocs.org/user-guide/custom-themes/#pageabs_url) or [`site_url`](https://www.mkdocs.org/user-guide/configuration/#site_url) for this?

I'm not familiar with implementing it in a jinja template. But we could maybe try something like this:

```JINJA
{% if page.abs_url in nav_item.url %}
    // Internal link
{% else %}
    // external link
{% endif %}
```

**Comment by squidfunk** [comment]
Yes, that's probably possible. However, we have to make sure that the examples stated in the docs still work.

**Comment by MrF3lix** [comment]
To make sure that the examples still work I suggest that we add another `elif` at the end to check if the nav item url is external and then show the external item. This way it will only be rendered if it doesn't have child elements, is not the main and is explicit external.

I will add a PR, maybe you could help me test it? I'm not sure where to beginn so a little input would be helpful :)

**Comment by squidfunk** [comment]
Jup, happy to collaborate on a PR.

**Comment by MrF3lix** [comment]
#1885

**Comment by squidfunk** [comment]
Fixed in cdb426d54e3cd9f4950feabdee4d6cb0a2f209b6.

**Comment by squidfunk** [comment]
Released as part of 5.5.9
